toolchest_client/files/unpack.py

```diff
@@ -43,26 +43,17 @@ def unpack_files(file_path_to_unpack, output_type):
         # Flat text needs no post-processing
         return file_path_to_unpack
     elif output_type == OutputType.GZ_TAR:
-        # The underlying shutil code does some, but it's limited
         unpacked_outputs_final_destination = os.path.dirname(file_path_to_unpack)
-        # Make sure the directory is empty except for the GZ_TAR
-        visible_files_in_directory = [
-            f for f in os.listdir(unpacked_outputs_final_destination) if not f.startswith('.')
-        ]
-        if len(visible_files_in_directory) > 1:
-            raise ValueError(
-                f"{unpacked_outputs_final_destination} is not empty. Please use an empty directory for output!"
-            )
```

Comment:
Instead of throwing an error while unpacking, we now print a warning in the beginning if there are any files which might conflict / be overwritten